### PR TITLE
add option for raspbee serial

### PR DESCRIPTION
**Pull request by frederic34**
need to add test in serialprober
https://github.com/WebThingsIO/serial-prober-node/pull/10

### CODE REVIEW

**Comment by frederic34** [comment]
image

File: index.js [hunk 1]
```diff
@@ -24,6 +24,9 @@ const XBEE_FTDI_FILTER = {
 
 const xbeeSerialProber = new SerialProber({
   name: 'XBee',
+
+  allowRpiAmaSerial: false,
```
Comment (mrstegeman) [inline index.js:28]: As mentioned in the other PR, please rename to `allowAMASerial`

**Review by mrstegeman — CHANGES_REQUESTED** [review]

**Comment by mrstegeman** [comment]
Code looks good, you just need to update the serial-prober dependency.

**Review by mrstegeman — APPROVED** [review]

**Comment by frederic34** [comment]
thank's for your help

**Pull request merged**